portfolio/src/test/java/com/google/sps/servlets/MessageTest.java

```diff
@@ -0,0 +1,53 @@
+// Copyright 2019 Google LLC
```

Comment:
Remind me to reach out and confirm on this. Should this be 2020? It might matter depending on what things Google intends with this.